vulture_os/system/config/api.py

```diff
@@ -78,11 +78,11 @@ def put(self, request, object_id):
         }, status=500)
 
     @api_need_key('cluster_api_key')
-    def patch(self, request, object_id):
+    def patch(self, request, object_id=None):
```

Comment:
method needs to apply to an existing object

```suggestion
    def patch(self, request, object_id):
```

Reply:
Same as before, object_id is never used in config_edit()